.eslintrc.json

```diff
@@ -4,7 +4,33 @@
     {
       "files": ["**/*.tsx", "**/*.ts"],
       "rules": {
-        "@typescript-eslint/explicit-function-return-type": "off"
+        "@typescript-eslint/brace-style": ["error"],
+        "@typescript-eslint/explicit-function-return-type": "off",
+        "@typescript-eslint/func-call-spacing": ["error"],
+        "@typescript-eslint/member-ordering": ["warn", { "default": ["method", "constructor", "field"] }],
+        "@typescript-eslint/no-require-imports": ["error"],
+        "react/no-unused-prop-types": ["warn", { "skipShapeProps": true }],
+        "arrow-spacing": "warn",
+        "array-bracket-spacing": ["warn", "never"],
+        "brace-style": "off",
```

Comment:
Why there is typescript rule `brace-style` is `error` above but global `brace-style` is `off` here